app/uk/gov/hmrc/brm/controllers/BirthEventsController.scala

```diff
@@ -65,40 +65,39 @@ trait BirthEventsController extends HeaderValidator with BRMBaseController {
               // does not exist on request
               countryAuditor.audit(Map("country" -> "no country specified"), None)
           }
-
           info(CLASS_NAME, "post()", s"error parsing request body as [Payload]")
           Future.successful(respond(BadRequest("")))
         },
         implicit payload => {
-
-          implicit val metrics : BRMMetrics = MetricsFactory.getMetrics()
-          implicit val auditor : BRMAudit = auditFactory.getAuditor()
-          implicit val features = FeatureFactory
-
-          if(!features().validate) {
-            //TODO: Need to work out
-            //TODO: 1. What We're going to log
-            //TODO: 2. Do we want to do any auditing within the FeatureFactory
-            //TODO: 3. Re-insert auditTransaction (see method below)
-            // auditTransaction()
-            info(CLASS_NAME, "post()", s"date of birth is before valid date / search is witched off (all/reference/details)")
-            Future.successful(respond(Ok(Json.toJson(BirthResponseBuilder.withNoMatch()))))
+          if(!BRMFormat.validBirthReferenceNumber(payload.whereBirthRegistered, payload.birthReferenceNumber)) {
+            Future.successful(respond(BadRequest("")))
```

Comment:
@manishsw @chris-ianson @anujakhare I think we should respond with an errors object as per guidance so they can understand the issue for each.